crates/biome_package/src/node_js_package/package_json.rs

```diff
@@ -36,7 +36,11 @@ pub struct PackageJson {
     pub optional_dependencies: Dependencies,
     pub license: Option<(Box<str>, TextRange)>,
 
-    pub(crate) raw_json: JsonObject,
+    pub author: Option<JsonValue>,
+    pub exports: Option<JsonValue>,
+    pub imports: Option<JsonValue>,
+    pub main: Option<JsonValue>,
```

Comment:
```suggestion
    pub main: Option<String>,
```

This can be safely be a String https://docs.npmjs.com/cli/v10/configuring-npm/package-json?v=true#main